docs/asyncapi.yaml

```diff
@@ -0,0 +1,92 @@
+asyncapi: 3.0.0
+info:
+  title: FCP-FD-COMMS
+  version: 1.0.0
+  description: Receive a communication message and route to appropriate channels
+
+channels:
+  sourceSystem-fcp-fd-comms:
+    address: sourceSystem-fcp-fd-comms
+    description: |
+      Accepts POST requests for communication notifications.
+    messages:
+      subscribe.message.0:
+        $ref: '#/components/messages/notifyMessage'
+
+operations:
+  SourceSystem/fcp-fd-comms.subscribe:
+    action: receive
+    channel:
+      $ref: '#/channels/sourceSystem-fcp-fd-comms'
+
+components:
+  messages:
+    notifyMessage:
+      title: gov.notify message
+      contentType: application/json
+      payload:
+        $ref: '#/components/schemas/NotificationPayload'
+
+  schemas:
+    NotificationPayload:
+      type: object
+      properties:
```

Comment:
These properties don't align to the CloudEvents.io schema.  Suggest adding a `data` property and push these items down to that.  Perhaps it would be good to break data into a separate schema and reference it there?

As for the CloudEvents.io spec, you can view the latest version here -> https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md 

There's a handy example at the bottom that I'd start with as it helps give you an idea of what the spec is.  Then when you're drafting the fields the additional context needed is above.